.circleci/config.yml

```diff
@@ -0,0 +1,146 @@
+version: 2.1
+
+executors:
+  test-executor:
+    machine:
+      image: ubuntu-1604:202004-01
```

Comment:
Are you sure ubuntu 16 is required?

Reply:
Nope. Its just the lastest VM image CircleCI has